Samples/Samples/ViewModel/CalendarViewModel.cs

```diff
@@ -161,38 +159,6 @@ void OnEndTimeSelected(object parameter)
 
         void OnChangeRequestCalendarSpecificEvents(string calendarId = null, DateTime? startDateTime = null, DateTime? endDateTime = null) => RefreshEventList(calendarId, startDateTime, endDateTime);
 
-        async void OnRequestCalendarWriteAccess()
-        {
-            try
-            {
-                await Calendar.RequestCalendarWriteAccess();
-            }
-            catch (PermissionException ex)
-            {
-                await DisplayAlertAsync($"Unable to request calendar write access: {ex.Message}");
-            }
-            catch (Exception ex)
-            {
-                await DisplayAlertAsync($"Unable to request calendar write access: {ex.Message}");
-            }
-        }
-
-        async void OnRequestCalendarReadAccess()
```

Comment:
Don't we still need the ability to request read access?

Reply:
But we removed the ability to request read access as a function from the api to assimilate with other apis?

They follow a methodology of if you make a request it checks the permissions at a per-request basis if access is unknown it makes a request for the required access at that time.